README.rst

```diff
@@ -67,3 +69,30 @@ via include list.
 This state will start the template service and has a depency on ``template.config``
```

Comment:
s/depency/dependency

Reply:
@daks Good catch!  This has been a long PR...

Reply:
@daks Wasn't my fault, that was based on existing content!

Reply:
Fixed.